code/game/objects/items/weapons/twohanded.dm

```diff
@@ -412,3 +412,15 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/rock
+	name = "rock"
+	desc = "If there were two miners were on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
+	force_unwielded = 6
+	force_wielded = 12
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "slag"
```

Comment:
"If there were two miners _were_ on the mining station ... "
should be "If there were two miners on the mining station ... " 
no?